client-data/board.html

```diff
@@ -39,7 +39,9 @@
 		<div id="menuItems">
 			<ul id="tools" class="tools">
 				<li class="tool" tabindex="-1">
-					<img class="tool-icon" width="35" height="35" src="" alt="icon" /><span class="tool-name"></span>
+					<img class="tool-icon" width="35" height="35" src="" alt="icon" />
+					<span class="tool-name"></span>
+					<img class="tool-icon secondaryIcon" width="35" height="35" src="" alt="icon" style='display:none;'/>
```

Comment:
```suggestion
					<img class="tool-icon secondaryIcon" width="35" height="35" src="" alt="icon" />
```

We can set the style in css